docs/changelog.md

```diff
@@ -14,6 +14,9 @@
 ### Breaking Changes
 * The `ContainerCreationConfigBuilder` constructor no longer takes in a test volume mountpoint
 * Added a `ContainerCreationConfigBuilder.WithTestVolumeMountpoint` for specifying the test volume mountpoint, which should be used instead if the default test volume mountpoint of `/kurtosis-test-volume` isn't acceptable
+* Replaced the argument `ContainerConfigFactory` in `AddService`and `AddServiceToPartition`with two arguments `ContainerCreationConfig`and an anonymous function which should returns `ContainerRunConfig`type;
```

Comment:
@leoporoli heads up that this is getting entered into `0.5.0`'s changelog, and should be moved to TBD